### ISSUE TITLE
[iOS] StructureToPtr_Flat_And_Nested_NonBlittableStructure_Success fails in System.Runtime.InteropServices.UnitTests

### ISSUE THREAD

**Issue by steveisok** [dotnet/runtime#73008]
[StructureToPtr_Flat_And_Nested_NonBlittableStructure_Success](https://github.com/dotnet/runtime/blob/aac729ff906a31f327823587748687c0308a4043/src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/Marshal/StructureToPtrTests.cs#L261) fails when using `Marshal.StructureToPtr` fails in AOT mode with the following message:

```
System.ExecutionEngineException: Attempting to JIT compile method '(wrapper other) void System.Runtime.InteropServices.Tests.PtrToStructureTests/SequentialClass:StructureToPtr (object,intptr,bool)' while running in aot-only mode. See https://docs.microsoft.com/xamarin/ios/internals/limitations for more information.

   at System.Runtime.InteropServices.Marshal.StructureToPtr[SequentialClass](SequentialClass structure, IntPtr ptr, Boolean fDeleteOld)
```

The test is being skipped for now.

**Comment by steveisok** [comment]
@vargaz These are still failing because it's trying to JIT:

https://github.com/dotnet/runtime/blob/aac729ff906a31f327823587748687c0308a4043/src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/Marshal/PtrToStructureTests.cs#L153-L160

https://github.com/dotnet/runtime/blob/aac729ff906a31f327823587748687c0308a4043/src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/Marshal/PtrToStructureTests.cs#L164-L171

https://github.com/dotnet/runtime/blob/aac729ff906a31f327823587748687c0308a4043/src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/Marshal/StructureToPtrTests.cs#L261-L282

**Comment by steveisok** [comment]
> 

No, it was not supposed to be closed.  There's one test left that is failing.

**Comment by MichalStrehovsky** [comment]
> Interesting, it looks like it was closed because it had the Fixes https://github.com/dotnet/runtime/issues/73008 in the commit message and since @MichalStrehovsky has write permissions it got closed when the commit was merged into the default branch of https://github.com/bflattened/runtime/pull/12

Wow, thanks Github :(. Sorry about that.
